earthpy/clip.py

```diff
@@ -119,7 +119,7 @@ def clip_shp(shp, clip_obj):
                              geometry column""")
 
 
-    if not any(shp.intersects(clip_obj)):
+    if not any(shp.intersects(clip_obj.unary_union)):
```

Comment:
This was a fix that was necessary for the function to work when clipping spatial points by a spatial polygon (the example in `docs/spatial-vector.rst` where Colorado glacier point locations are clipped by the boundary of Rocky Mountain National Park).

Reply:
oh gosh good catch. yes you'd need unary_union to ensure that the full extents overlap. thank you!!

Reply:
:+1: